openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -231,10 +277,12 @@ def test_get_cohort_with_assign(self):
         # get_cohort should return a group for user
         self.assertEquals(cohorts.get_cohort(user, course.id).name, "AutoGroup")
 
-
-    def test_auto_cohorting(self):
+    def test_cohorting_with_auto_cohort_groups(self):
         """
-        Make sure cohorts.get_cohort() does the right thing with auto_cohort_groups
+        Make sure cohorts.get_cohort() does the right thing with auto_cohort_groups.
+        If there are auto cohort groups then a user should be assigned one.
+        Also verifies that cohort config changes on studio/moduletore side will
+        not be reflected on lms after the migrations are done.
         """
```

Comment:
this is great coverage, but i think it might be better to test the two things being checked here separately.